pkg/cli/admin/prune/images/images.go

```diff
@@ -221,6 +217,8 @@ func (o *PruneImagesOptions) Complete(f kcmdutil.Factory, cmd *cobra.Command, ar
 	if len(o.ClientConfig.BearerToken) == 0 {
 		return errNoToken
 	}
+	o.ClientConfig.QPS = 100
```

Comment:
@soltysh is there a way to do it better?

Reply:
Nope, there is none. This is fine.